### ISSUE TITLE
Wrapper or container for template

### ISSUE THREAD

**Issue by Mramorov** [CrocoBlock/suggestions#6583]
Hello,
I noticed that you have several widgets (Tabs, Accordion, Unfold) that can use the template as content. However, I didn't find a mechanism that would allow me to simply insert the template on the page (not Magic Button which just copies template), that is, a simple wrapper for the template. In order not to add an extra widget, then, as an option, it would be possible to modify the 'Unfold' widget by adding the option to disable the folding mechanism, so that the widget works in this case simply as a container for the template.

**Comment by aalexandr21** [comment]
Yes, you can use the Expand widget if you prefer and hide the Collapse button using your own CSS.

Also, we added to our backlog this issue, which is able to display some templates via a widget
